packages/@ama-sdk/generator-sdk/src/generators/java-client-core/templates/swagger-codegen-java-client/src/main/resources/javaClient/settings.mustache

```diff
@@ -3,17 +3,17 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <mirrors>
     <mirror>
-      <id>public-artifactory</id>
+      <id>des-sdks</id>
       <url>https://pkgs.dev.azure.com/AmadeusDigitalAirline/DES-SDKs/_packaging/des-sdks/maven/v1</url>
-      <name>External Repository</name>
-      <mirrorOf>public-artifactory</mirrorOf>
+      <name>External Azure Artifact repository</name>
+      <mirrorOf>des-sdks</mirrorOf>
     </mirror>
   </mirrors>
   <servers>
     <server>
-      <id>public-artifactory</id>
-      <username>${env.AR_USER}</username>
-      <password>${env.AR_PASSWORD}</password>
+      <id>des-sdks</id>
+      <username>AmadeusDigitalAirline</username>
```

Comment:
Why using static string instead of variable?

Reply:
It's just the cascading 
maybe we need to fix it on v8 if you are not agree with what @mrednic-1A has done

Reply:
It's coming from a fix done when passing from jfrog to azure. we can redo it  8.0 if needed